index.ts

```diff
@@ -1,3 +1,19 @@
 import cli from "./cli";
 
+import Scrubber from "./scrubber/scrubber";
+import { ScrubberAction } from "./scrubber/scrubberTypes";
+
+async function scrub() {
+  try {
+    const actions: ScrubberAction[] = [{ type: "remove", tags: ["@remove"] }];
+    const scrubber = new Scrubber();
+
+    await scrubber.parseConfig("scrubber/scrubberConfig.json");
+    await scrubber.start(actions);
```

Comment:
To confirm my understanding, default tag settings go in the config file, and results from CLI user prompts are passed as the `actions` argument?

Reply:
Yep, that is how it is currently set up.
Alternatively we can just pass all tags through the function and not put any in the config. Thoughs? @alexguo8